src/modal/modal.ts

```diff
@@ -0,0 +1,26 @@
+import {Injectable, TemplateRef} from '@angular/core';
+
+import {NgbModalStack} from './modal_stack';
+import {NgbModalRef} from './modal_ref';
+
+export interface NgbModalOptions {
+  backdrop?: boolean | 'static';
+  keyboard?: boolean;
+  size?: 'sm' | 'lg';
```

Comment:
The above is not documented neither here, nor in the `NgbModalWindow` → no documentation on the demo site